delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/jsonld/JsonLdSourceParser.scala

```diff
@@ -61,11 +62,12 @@ trait JsonLdSourceParser {
     * @return a tuple with the resulting @id iri and the decode value
     */
   def decode[A: JsonLdDecoder, R](project: Project, iri: Iri, source: Json)(implicit
```

Comment:
if you pass here as implicit the `ResolverResolution`, you don't need to have the extra line

```scala
implicit val rcr: RemoteContextResolution = contextResolution(projectRef)
```
in every implementation. You can just do it directly here, since you already have the project too